fix: App crashing on adding picture at Upload

CloudyPadmal opened the pull request:
The query handler used which is **managedQuery()** is deprecated! Replaced it with _**getContentResolver().query()**_

Abhi2424shek commented on app/src/main/java/vn/mbm/phimp/me/PhotoSelect.java:
```diff
@@ -309,7 +309,7 @@ public void run() {
 	 			}
 	    	}catch(RuntimeException runex){
-	    		this.onCancelled();
+	    		//this.onCancelled();
```
Should this be uncommented, I guess it is a good UI to stop the process when an exception has occurred.

CloudyPadmal commented:
@Abhi2424shek I've uncommented it and made the PR again! I referred to this [link](http://stackoverflow.com/questions/24463364/android-activity-asynctask-and-oncancelled) when I commented it! But I agree to what you say! :D Can you review this?

CloudyPadmal commented:
@Abhi2424shek There were few other lines where the deprecated code needed to be replaced in the same class file! I've changed them and squashed them together!

Abhi2424shek commented on app/src/main/java/vn/mbm/phimp/me/PhotoSelect.java:
```diff
@@ -281,7 +281,7 @@ protected String doInBackground(Cursor... cursor) {
 	 				boolean check = PhimpMe.cache.check(path);
 	 				if(!check){	 					
 	 					@SuppressWarnings("deprecation")
```
Since you removed the depreciated code can you remove the supress tag as well as we don't need to supress the depreciation as there w9uld be no depreciation!

CloudyPadmal replied:
Sure @Abhi2424shek !! PR is right away :D

CloudyPadmal commented:
@Abhi2424shek Thank you!! :1st_place_medal:
